server/tserver/src/main/java/org/apache/accumulo/tserver/NativeMap.java

```diff
@@ -70,11 +70,10 @@
   static {
     // Check standard directories
     List<File> directories = new ArrayList<>(Arrays.asList(new File[] {new File("/usr/lib64"), new File("/usr/lib")}));
-    // Check in ACCUMULO_HOME location, too
-    String accumuloHome = System.getenv("ACCUMULO_HOME");
-    if (accumuloHome != null) {
-      directories.add(new File(accumuloHome + "/lib/native"));
-      directories.add(new File(accumuloHome + "/lib/native/map")); // old location, just in case somebody puts it here
```

Comment:
I think we should limit the ways users can set native libraries are loaded but fail if native maps are enabled and the libraries cannot be loaded.  Something like below:

1. Check if `tserver.memory.maps.native.enabled` is true in `accumulo-site.xml`. If false, do nothing.  
2. If true, check if the system property `accumulo.native.lib.dir` is set.  If not, warn and exit.
3. If the property is set, try to load native library and warn/exit if library cannot be loaded.